### PR TITLE
Build against latest version of prometheus golang-client

### PR DESCRIPTION
**Pull request by ese**
Resolved conflicts of #38

### CODE REVIEW

File: collector/mongod/global_lock.go [hunk 1]
```diff
@@ -11,18 +11,10 @@ var (
 		Name:      "ratio",
 		Help:      "The value of ratio displays the relationship between lockTime and totalTime. Low values indicate that operations have held the globalLock frequently for shorter periods of time. High values indicate that operations have held globalLock infrequently for longer periods of time",
 	})
-	globalLockTotal = prometheus.NewCounter(prometheus.CounterOpts{
-		Namespace: Namespace,
-		Subsystem: "global_lock",
-		Name:      "total",
-		Help:      "The value of totalTime represents the time, in microseconds, since the database last started and creation of the globalLock. This is roughly equivalent to total server uptime",
-	})
-	globalLockLockTotal = prometheus.NewCounter(prometheus.CounterOpts{
```
Comment (AlekSi) [inline collector/mongod/global_lock.go]: Why this metric was removed?

Comment (ese) [inline collector/mongod/global_lock.go]: Good catch. I guess it was a mistake so i'll add it again

**Comment by kinghrothgar** [comment]
Is this waiting on review or for the globalLockLockTotal metric to be added back?

**Comment by uoryon** [comment]
@ese can you add the globalLockLockTotal back now? we need this repo can build.

**Comment by ese** [comment]
@uoryon I added it here https://github.com/percona/mongodb_exporter/pull/63/commits/4a7f09119077aa8231b4e2dacb37952f1399d3e8

**Comment by uoryon** [comment]
@ese thx

**Comment by AlekSi** [comment]
Original issue solved in #67. More metrics are coming.

**Pull request closed without merging**